connectors/sources/azure_blob_storage.py

```diff
@@ -165,34 +162,28 @@ async def get_content(self, blob, timestamp=None, doit=None):
         Returns:
             dictionary: Content document with id, timestamp & text
         """
-        blob_size = int(blob["size"])
-        if not (doit and blob_size > 0):
+        file_size = int(blob["size"])
+        if not (doit and file_size > 0):
             return
 
-        blob_name = blob["title"]
-        file_extension = (os.path.splitext(blob_name)[-1]).lower()
-        if file_extension not in TIKA_SUPPORTED_FILETYPES:
-            self._logger.warning(f"{blob_name} can't be extracted")
-            return
+        filename = blob["title"]
+        file_extension = self.get_file_extension(filename).lower()
 
-        if blob["tier"] == "Archive":
-            self._logger.warning(
-                f"{blob_name} can't be downloaded as blob tier is archive"
-            )
+        if not self.can_file_be_downloaded(file_extension, filename, file_size):
             return
 
-        if blob_size > DEFAULT_FILE_SIZE_LIMIT:
+        if blob["tier"] == "Archive":
             self._logger.warning(
-                f"File size {blob_size} of file {blob_name} is larger than {DEFAULT_FILE_SIZE_LIMIT} bytes. Discarding the file content"
+                f"{filename} can't be downloaded as blob tier is archive"
             )
             return
```

Comment:
This can probably go upper on line 168 - that's just more from aesthetic side :)

Reply:
Aesthetics are good, I'll move this